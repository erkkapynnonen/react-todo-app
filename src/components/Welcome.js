import React, {Component} from 'react'

export default class Welcome extends Component {
  constructor(props) {
    super(props);
    this.state = {
      appRepositoryUrl: 'https://github.com/erkkapynnonen/react-todo-app'
    };

  }
  render() {
    return (
      <div className="row">
        <div className="col-xs-12 col-md-6">
          <h1>Welcome!</h1>
          <p>This is a simple to-do app made with React for demo and learning purposes.</p>
          <p>Check out the repo on <a target="_blank" href={this.state.appRepositoryUrl}>GitHub</a>.</p>
          <p>You can use the demo account demo/demo for login as well as you Google/FB account via Oauth, provided by Auth0.</p>
        </div>
        <div className="col-xs-12 col-md-6">
          <h3>Authenticate!</h3>
        </div>
      </div>
    )
  }
}
