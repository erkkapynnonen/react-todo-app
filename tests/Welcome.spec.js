import React from 'react';
import renderer from 'react-test-renderer';
import Welcome from '../src/components/Welcome';

describe('Welcome component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Welcome />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
