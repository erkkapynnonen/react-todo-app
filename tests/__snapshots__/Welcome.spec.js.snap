// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component renders correctly 1`] = `
<div>
  <h1>
    Welcome!
  </h1>
  <p>
    This is a simple to-do app made with React for demo and learning purposes.
  </p>
  <p>
    Check out the repo on 
    <a
      href="https://github.com/erkkapynnonen/react-todo-app"
      target="_blank"
    >
      GitHub
    </a>
    .
  </p>
  <p>
    You can use the demo account demo/demo for login as well as you Google/FB account via Oauth, provided by Auth0.
  </p>
</div>
`;
